indexer-app/src/cmr/indexer/data/index_set.clj

```diff
@@ -321,7 +321,7 @@
           :archive-center.lowercase       m/string-field-mapping
           :data-center                    (m/stored m/string-field-mapping)
           :data-center.lowercase          m/string-field-mapping
-          :spatial-keyword                m/string-field-mapping
+          :spatial-keyword                (m/doc-values m/string-field-mapping)
```

Comment:
You cannot change mappings for a field in an existing Elasticsearch index (maybe we should add a test for this similar to the granule index one).

If these are collection fields it is ok that they aren't doc-values (collections use a tiny amount of memory in Elasticsearch when compared to granules).